test_common/page_objects/claimants_details_page.rb

```diff
@@ -6,146 +6,179 @@ class ClaimantsDetailsPage < BasePage
+          choose(factory_translate(user_persona.agree_with_employment_dates), name: 'claimants_detail[agree_with_employment_dates]')
+          if (t(user_persona.agree_with_employment_dates) == t('questions.agree_with_employment_dates.no.label'))
+            # Employment started
+            day, month, year = user_persona.employment_start.split('/')
+            employment_start.day.set(day)
+            employment_start.month.set(month)
+            employment_start.year.set(year)
+            # Employment ended
+            day, month, year = user_persona.employment_end.split('/')
+            employment_end.day.set(day)
+            employment_end.month.set(month)
+            employment_end.year.set(year)
+            disagree_employment.set(user_persona.disagree_employment)
           end
         end
       end
 
+      # Is their employment continuing? (optional)
       section :continued_employment_question, :single_choice_option, 'questions.continued_employment.label', exact: false do
-        include SingleChoiceOptionSection
-
-        def set_for(user_persona)
-          choose(user_persona.continued_employment)
+        include ET3::Test::I18n
+        element :yes, :gds_multiple_choice_option, 'questions.continued_employment.yes.label' do
+          element :selector, :css, 'input[type="radio"]'
+          def set(*args); selector.set(*args); end
+        end
+        element :no, :gds_multiple_choice_option, 'questions.continued_employment.no.label' do
+          element :selector, :css, 'input[type="radio"]'
+          def set(*args); selector.set(*args); end
+        end
+        def set_for(value)
+          choose(factory_translate(value), name: 'claimants_detail[continued_employment]')
         end
       end
-
+      # Is the claimant's description of their job or job title correct? (optional)
       section :agree_with_claimants_description_of_job_or_title_question, :single_choice_option, 'questions.agree_with_claimants_description_of_job_or_title.label', exact: false do
-        include SingleChoiceOptionSection
-
+        include ET3::Test::I18n
+        element :yes, :gds_multiple_choice_option, 'questions.agree_with_claimants_description_of_job_or_title.yes.label' do
+          element :selector, :css, 'input[type="radio"]'
+          def set(*args); selector.set(*args); end
+        end
+        element :no, :gds_multiple_choice_option, 'questions.agree_with_claimants_description_of_job_or_title.no.label' do
+          element :selector, :css, 'input[type="radio"]'
+          def set(*args); selector.set(*args); end
+        end
         section :disagree_claimants_job_or_title, :textarea_labelled, 'questions.agree_with_claimants_description_of_job_or_title.disagree_claimants_job_or_title.label', exact: false do
-          delegate :set, to: :root_element
+          def set(*args); root_element.set(*args); end
```

Comment:
Why this change ?  The delegate methos is a standard way of doing this

Reply:
As @sivvoy has explained this is due to an issue with site_prism.